pkg/controller/clusterprovision/installlogmonitor.go

```diff
@@ -51,6 +52,16 @@ func (r *ReconcileClusterProvision) parseInstallLog(log *string, pLog log.FieldL
 		return unknownReason, regexBadMessage
 	}
 
+	additionalRegexesRaw, _ := regexCM.Data[additionalRegexDataEntryName]
+
+	additionalRegexes := []installLogRegex{}
+	if additionalRegexesRaw != "" {
+		if err := yaml.Unmarshal([]byte(additionalRegexesRaw), &additionalRegexes); err != nil {
+			pLog.WithError(err).Errorf("cannot unmarshal data from %s configmap", regexConfigMapName)
```

Comment:
Lets clarify in the error message that it's something specific to the additional regexes key.